app/controllers/neighborhoods_controller.rb

```diff
@@ -0,0 +1,41 @@
+class NeighborhoodsController < ApplicationController
+	def index
+		@neighborhoods = Neighborhood.all
+	end
+
+	def new
+		@neighborhood = Neighborhood.new
+	end
+
+	def contains_title?(title)
+		@neighborhoods = Neighborhood.all
+		@neighborhoods.each do |item|
+			if item.title == title
+				return true
+			end
+		end
+		return false	
+	end
+
+	def create
+		@neighborhood = Neighborhood.new
+		@repeat_error = contains_title?(neighhorhood_params[:title])
```

Comment:
http://guides.rubyonrails.org/active_record_validations.html#uniqueness

@HacksForHugs This is what you'll need. We can talk about how this works. Ping me if you need help!